src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin text-styles($font-weight, $font-size, $line-height, $color) {
+  font-weight: $font-weight;
+  font-size: $font-size;
+  line-height: $line-height;
+  color: $color;
+}
+
+@mixin image-size($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+@mixin flex($justify-content, $align-items) {
```

Comment:
No need to create a mixin for all your code, but you had a good practice, good job 👍